app/templates/components/billing/payment.hbs

```diff
@@ -77,11 +77,27 @@
         <p>{{stripeError.message}}</p>
       {{/if}}
     </StripeCard>
+    <TravisForm as |form|>
+      <form.field
+        @disableFrame={{true}}
+        @value={{this.isDisclaimerChecked}}
+        @showValidationStatusIcons={{false}}
+        @onChange={{action (toggle 'isDisclaimerChecked' this)}} 
+        as |field|
+      >
+      <div class="flex">
+        <field.checkbox @name='disclaimer' class="flex-1" data-test-disclaimer-checkbox='true'/>
+        <span class="text-xs">
+          I authorize Travis CI to send instructions to the financial institution that issued my card to take payments from my card account in accordance with the terms of my agreement with you.
```

Comment:
Would it make sense to link the terms here? Also I'm not sure about the copy, it sounds too personal to me. @sorta @Pezi777, what do you think?

Reply:
I think linking to the terms makes sense to me :+1: And yeah, I agree about the copy, something about it sounds somewhat informal/personal. Maybe it's the bit at the end?

Reply:
Thank you, @artursmirnov and @sorta, for ideas! @emmanuel-ci right. We can add the text: `I authorize Travis CI to send instructions to the financial institution that issued my card to take payments from my card account in accordance with the terms of my agreement with you.` to https://docs.travis-ci.com/legal/terms-of-service/ under `Payments` section and simplify the in UI as suggested above.